.github/workflows/test.yml

```diff
@@ -36,9 +50,16 @@ jobs:
           python -m pip install --upgrade pip wheel
           python -m pip install --upgrade --upgrade-strategy=eager tox
 
+      - name: Set TMPDIR to C:\tmp on Windows
+        # To avoid <https://github.com/conda/conda/issues/10501>
+        if: startsWith(matrix.os, 'windows')
+        run: |
+          mkdir -p /c/tmp
+          printf 'TMPDIR=%s\n' 'C:\tmp' >> "$GITHUB_ENV"
```

Comment:
since `./run-tests.sh` is used on all OSes, and already overrides TMPDIR for OSX, shouldn't it just do the same for windows there so we do not spread this customization around?